libs/application/templates/driving-license/src/lib/drivingLicenseTemplate.ts

```diff
@@ -63,6 +73,26 @@ const template: ApplicationTemplate<
               },
               write: 'all',
               delete: true,
+              api: [
+                NationalRegistryUserApi,
+                TeachersApi,
+                UserProfileApi,
+                PaymentCatalogApi.configure({
+                  params: { orginizationId: '6509142520' },
```

Comment:
`libs/application/templates/passport/src/lib/PassportTemplate.ts` stores the organization id in constants where it is easier to find and change.
Consider doing the same here?